### PR TITLE
Create 'Users' section of Controller UG

### PR DESCRIPTION
**Pull request by ianf77**
Added files and images

Create section 9 of the Controller User Guide

https://issues.redhat.com/browse/AAP-15769

### CODE REVIEW

File: downstream/modules/platform/proc-controller-add-organization-user.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@ To add a user or team to an organization, the user or team must already exist.
 To add existing users or team to the Organization:
 
 .Procedure
-. In the *Access tab*, click btn:[Add].
+. In the *Access tab* of the *Oraganization* page, click btn:[Add].
```
Comment (EMcWhinn) [inline downstream/modules/platform/proc-controller-add-organization-user.adoc:10]: "Organization"

File: downstream/modules/platform/proc-controller-creating-a-user.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
+[id="proc-controller-creating-a-user"]
+
+= Creating a user
+
+To create a new user:
+
+.Procedure
+. On the *Users* page, click the *Add* button.
```
Comment (EMcWhinn) [inline downstream/modules/platform/proc-controller-creating-a-user.adoc:8]: Suggested re-write "...click btn[Add]."

File: downstream/modules/platform/proc-controller-creating-a-user.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+[NOTE]
+====
+If the user is not newly-created, the details screen displays the last login activity of that user.
+
+//image:users-last-login-info.png[image]
+====
+
+If you log in as yourself, and view the details of your user profile, you can manage tokens from your user profile. 
+//For more information, see xref:proc-controller-users-tokens[Adding a user token] for more detail.
```
Comment (EMcWhinn) [inline downstream/modules/platform/proc-controller-creating-a-user.adoc:53]: Remove "for more detail."

File: downstream/modules/platform/proc-controller-deleting-a-user.adoc [hunk 4]
```diff
@@ -0,0 +1,16 @@
+[id="proc-controller-deleting-a-user"]
+
+= Deleting a user
+
+Before you can delete a user, you must have user permissions. 
+When you delete a user account, the name and email of the user are permanently removed from {ControllerName}.
+
+.Procedure
+. In the navigation panel, expand the *Access*.
```
Comment (EMcWhinn) [inline downstream/modules/platform/proc-controller-deleting-a-user.adoc:9]: The steps in the guide are currently inconsistent, maybe it is something I can correct in the whole guide before we publish. After editorial review and confirming accessibility options with Donna, I re-wrote these as:

. From the navigation panel, select menu:Access[Users].

File: downstream/modules/platform/proc-controller-user-permissions.adoc [hunk 5]
```diff
@@ -0,0 +1,36 @@
+[id="proc-controller-user-permissions"]
+
+= Adding and removing user permissions
+
+To add permissions to a particular user:
+
+.Procedure
+. From the *Users > Details* page, click btn:[Add], which opens the *Add Permissions* wizard.
```
Comment (EMcWhinn) [inline downstream/modules/platform/proc-controller-user-permissions.adoc:8]: Suggested re-write:

. From the *Users* list view, click on the desired user.
. On the *Details* tab click btn:[Add]. This opens the *Add Permissions* wizard.

File: downstream/modules/platform/proc-controller-user-tokens.adoc [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+To create a token for your user:
+
+.Procedure
+. Select your user from the *Users* list view to configure your OAuth 2 tokens.
+. Select the *Tokens* tab from your user's profile.
+. Click btn:[Add] to open the *Create Token* window.
+. Enter the following information:
+* *Application*: Enter the name of the application with which you want to associate your token. 
++
+Alternatively, you can search for it by clicking the image:search.png[Search,15,15] icon.
```
Comment (EMcWhinn) [inline downstream/modules/platform/proc-controller-user-tokens.adoc:20]: In the editorial review I was advised to follow the procedures section of IBM SG when there was more than one option:
https://www.ibm.com/docs/en/ibm-style?topic=format-procedures

This procedure would start like:

" 1. Create a token for your user by using one of these methods:
       *Select your user...
       
I'm not sure how best to structure this now though.

Comment (ianf77) [reply downstream/modules/platform/proc-controller-user-tokens.adoc:20]: There aren't two methods, there are two options for *Application* either add the name, or search for it.

**Review by EMcWhinn — APPROVED** [review]
LGTM

**Pull request merged**